src/ui/Base/CoveoJQuery.ts

```diff
@@ -18,14 +18,19 @@ if (jQueryIsDefined()) {
 }
 
 export function initCoveoJQuery() {
-  jQueryInstance = window['$'];
+  if (window['$']) {
+    jQueryInstance = window['$']
+  } else {
+    jQueryInstance = window['Coveo']['$']
```

Comment:
This function is called only if the check jQueryIsDefined is true which verifies this.
`return window['Coveo'] != undefined && window['Coveo']['$'] != undefined;`
I refactored to try to make this clearer.